app/src/main/java/com/yeonae/chamelezone/view/home/adapter/HomePlaceRvAdapter.kt

```diff
@@ -116,10 +113,15 @@ class HomePlaceRvAdapter(val currentLatitude: Double, val currentLongitude: Doub
     }
 
     fun itemChange(position: Int) {
-//        notifyItemChanged(placeList.lastIndex, 1)
         notifyItemChanged(position, 1)
     }
 
+    fun addDataDistance(lat: Double, log: Double) {
```

Comment:
```suggestion
    fun replaceDataDistance(lat: Double, log: Double) {
```
사용자 이동에 따라 다시 실행 될 수 도 있는 함수이니 add보다는 다른 네이밍이 의미를 잘 나타낼 것 같아요

Reply:
수정했습니다
2a3f591